.circleci/config.yml

```diff
@@ -358,6 +360,7 @@ jobs:
       - run_pytest:
           with_sudo: true
           split: true
+          pytest_options: --reruns 3
```

Comment:
Would maybe set this to at most 2 so that failing tests don't make it waste too much time to fail.  If a test fails more than twice in a row it should be fixed to be more robust.